.github/CODEOWNERS

```diff
@@ -217,11 +217,14 @@
 # ServiceLabel: %Cognitive - Face
 # ServiceOwners:                                                   @dipidoo @longli0 @ShaoAnLin @leareai @Han-msft
 
+# PRLabel: %Cognitive - Content Understanding
+/sdk/contentunderstanding/                                         @vkurpad @yungshinlintw @bojunehsu @changjian-wang
+
 # PRLabel: %Cognitive - Form Recognizer
-/sdk/documentintelligence/                                         @vkurpad
+/sdk/documentintelligence/                                         @vkurpad @yungshinlintw @bojunehsu
```

Comment:
- `vkurpad` is missing the public membership to the `Microsoft` organization and write permissions.